pkg/kubelet/kubelet_pods.go

```
@@ -474,11 +473,17 @@ func (kl *Kubelet) GenerateRunContainerOptions(pod *v1.Pod, container *v1.Contai
 			glog.Errorf("Error on creating %q: %v", p, err)
 		} else {
 			opts.PodContainerDir = p
+			if err := kl.chownDirForRemappedIDs(p); err != nil {
+				glog.Errorf("Error while chowning %s", p)
```

shouldnt this return an error rather than log an error?